openpype/hosts/nuke/api/plugin.py

```diff
@@ -316,8 +314,8 @@ def create(self, subset_name, instance_data, pre_create_data):
         # make sure subset name is unique
         if self.check_existing_subset(subset_name, instance_data):
             raise NukeCreatorError(
-                ("subset {} is already published"
-                 "definition.").format(subset_name))
+                ("Subset '{}' is already created "
+                 "in nodes! Change variant name!").format(subset_name))
```

Comment:
I'd rephrase this as: `"A publish instance for '{subset}' in asset '{asset}' already exists in the scene. Please change the variant name to ensure unique output"`.

Reply:
I had alternate it a bit, but yeah agree it sounded bit harsh. Also, I had reworked it completely because in fact we didn't need to have exceptions per plugin, but rather abstracted it into the function (I believe someone had already mentioned it so there it is.)